src/index.js

```diff
@@ -1 +1,41 @@
 import './style.css';
+import getScore from './modules/getScore.js';
+import postScore from './modules/postScore.js';
+
+let array = [];
+const playersTable = document.querySelector('#displayPlayers');
+const refresh = document.querySelector('#refresh');
+const postData = document.querySelector('form');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#scoreNum');
+const warnignMessage = document.querySelector('#warning');
+
+const updateBoard = async () => {
+  array = await getScore();
+  playersTable.innerHTML = '';
+  array.forEach((element) => {
```

Comment:
- [**Optional**] For a great display, we can even sort the scores on the board :+1: 

 ` array.sort((a,b)=> a.score - b.score).forEach((element) => {`